src/compounds/product-table/src/components/data-text-subtext.tsx

```diff
@@ -0,0 +1,29 @@
+/** @jsx jsx */
```

Comment:
I think the name of this formatted is a little ambiguous. My initial thought was that it was text with some subtext underneath it, rather than text followed by smaller text (on the same line). Is it worth naming this `data-text-subscript`, and creating a new formatted called `data-text-subtext` which puts smaller text underneath?

Reply:
Or maybe a more generic formatter called `data-text`, which simply adds text but allows for `subtext` to be supplied?

Reply:
You're right! I misread your messages yesterday.